### DISCUSSION TITLE
Using pin 7 for led

### DISCUSSION THREAD

**Discussion by ericsch333** [bassamanator/Sovol-SV06-firmware#67] [Ideas]
How do you add the code to use pin 7 for an led in klipper ? Can it be setup like the runout ?

**Comment by bassamanator** [comment]
I saw this done somewhere. You don't need any code to make this happen from what I remember, the led should just turn on when connected.

**Comment by ericsch333** [comment]
It seems like the pin is disabled I tired it and no power I also tested
with a meter and nothing. I have checked Google and it seem that it hasn't
worked on klipper for anyone

**Comment by ericsch333** [comment]
Had to run power to it it is disconnected in the control box
![16876430420441155102359672906237](https://github.com/bassamanator/Sovol-SV06-firmware/assets/563796/d882282e-bed5-4c98-b45e-11e7cdf5ecc5)

**Comment by bassamanator** [comment]
Nice.

What are the steps to make this work?

**Comment by ericsch333** [comment]
There is a extra wire in the control box for the runout sensor cut the end
off and add it to the 24v fan connector on the power block. It's labeled
Def

**Comment by bassamanator** [comment]
Ah of course. At some point I may include this on the everything sovol repo. Thanks!

**Comment by ericsch333** [comment]
I am going to buy you some coffee I thank you for your hard work and your
the man
